android/modules/contacts/src/java/ti/modules/titanium/contacts/CommonContactsApi.java

```diff
@@ -143,8 +143,6 @@ protected static String getImTextType(int type)
 			key = "custom";
 		} else if (type == ContactsContract.CommonDataKinds.Im.PROTOCOL_MSN) {
 			key = "msn";
-		} else if (type == ContactsContract.CommonDataKinds.Im.PROTOCOL_YAHOO) {
```

Comment:
Should this be removed? Does Yahoo still have IM? I don't think it's related to our Ti.Yahoo API.